Northstar.CustomServers/mod/scripts/vscripts/lobby/_private_lobby.gnut

```diff
@@ -244,23 +244,21 @@ bool function ClientCommandCallback_PrivateMatchSetPlaylistVarOverride( entity p
 bool function ClientCommandCallback_ResetMatchSettingsToDefault( entity player, array<string> args )
 {
 	if ( GetConVarInt( "ns_private_match_only_host_can_change_settings" ) >= 1 )
-		if ( !NSIsPlayerIndexLocalPlayer( player.GetPlayerIndex() ) )
+		if ( !NSIsPlayerLocalPlayer( player ) )
 			return true
 	
-	PlayerChangedTheGame( player , " reset to default" , args )
+	LogPrivateMatchChange( player , " reset to default" , args )
 
 	ClearPlaylistVarOverrides()
 	return true
 }
 
-void function PlayerChangedTheGame( entity player , string step , array<string> args ){
-	if( step.find( "mode" ) || step.find( "map" )){
+void function LogPrivateMatchChange( entity player , string step , array<string> args )
+{
+	if( step.find( "mode" ) || step.find( "map" ) )
 		print( player.GetPlayerName() + step + args[ 0 ] + ".---" + "UID:" +player.GetUID() )
-	}
-	else if(step.find("setting")){
+	else if ( step.find( "setting" ) )
 		print( player.GetPlayerName() + step + args[ 0 ] + " to "+ args[ 1 ] + ".---" + "UID:" +player.GetUID() )
-	}
-	else{
+	else
 		print( player.GetPlayerName() + step + ".---" + "UID:" + player.GetUID())
-	}
 }
```

Comment:
@RoyalBlue1 cause I have no clue if our Squirrel style guide says to keep the curly brackets or not (I'd keep em personally)

Reply:
Just wanna point out that not adding brackets on single line branches is bad practice and I dislike. Anyway, merging as is :trollface: